doc/mainpage.dox

```diff
@@ -8,7 +8,6 @@ by an image processing system.  In particular:
    - \ref afwSecImage : Representation of Images (and Masks)
    - \ref afwSecPyImage : How to manipulate images from python
    - \ref afwSecDisplay : How to display images
-   - \ref afwSecDetection : Detection of sources in images
```

Comment:
I assume you built the Doxygen docs (`scons doc`) and made sure there were no other links to the deleted section?

Reply:
I did indeed; it successfully built and grep turned up no other links.